src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { render, screen } from '@testing-library/react';
+import { DataList } from '../../../../../core';
+import { AssetDataListItemStructure, type IAssetDataListItemStructureProps } from './assetDataListItemStructure';
+
+describe('<AssetDataListItemStructure /> component', () => {
+    const createTestComponent = (props?: Partial<IAssetDataListItemStructureProps>) => {
+        return (
+            <DataList.Root entityLabel="Assets">
+                <DataList.Container>
+                    <AssetDataListItemStructure {...props} />
+                </DataList.Container>
+            </DataList.Root>
+        );
+    };
+
+    it('renders tokenName, symbol, and the logoSrc', () => {
+        const props = {
+            logoSrc: 'https://assets.coingecko.com/coins/images/279/standard/ethereum.png?1696501628',
+            tokenName: 'Ethereum',
+            amount: 420.69,
+            symbol: 'ETH',
+            USDAmount: 1230000,
+            changedAmount: 420.69,
+            changedPercentage: 0.05,
```

Comment:
We don't need to pass all these properties if we're only testing the `tokenName` and `symbol` rendering. Same for the test below, only specify the props you're actually testing